### PR TITLE
Feat : add doc for the `@swarmion/nx-plugin`

### PR DESCRIPTION
**Pull request by guillaumeduboc**

### CODE REVIEW

File: user-docs/documentation/docs/code-structure/nx-plugin.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+---
+sidebar_position: 4
```
Comment (fargito) [inline user-docs/documentation/docs/code-structure/nx-plugin.md:2]:
```suggestion
sidebar_position: 5
```

Comment (fargito) [inline user-docs/documentation/docs/code-structure/nx-plugin.md:2]:
```suggestion
sidebar_position: 5
```

File: packages/nx-plugin/README.md [hunk 2]
```diff
@@ -4,8 +4,16 @@ Generate new libraries and services in your Serverless monorepo.
 
 This package is part of the [Swarmion](https://www.swarmion.dev) project. See its documentation for more insights.
 
-It is meant to be used on a repo structured like the [Swarmion template](https://github.com/swarmion/template)
+# Installation
```
Comment (fargito) [inline packages/nx-plugin/README.md:7]:
```suggestion
## Installation
```

File: user-docs/documentation/docs/code-structure/nx-plugin.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+---
+
+# Nx Plugin
+
+Generate new packages respecting the [monorepo code structure](./monorepo.md#the-monorepo-structure) in one command
```
Comment (fargito) [inline user-docs/documentation/docs/code-structure/nx-plugin.md:7]:
```suggestion
Generate new packages respecting the [monorepo code structure](./monorepo#the-monorepo-structure) in one command
```

**Review by fargito — CHANGES_REQUESTED** [review]
Can you:
- add documentation on the `--directory` option for both service and library generators?
- add a link to your documentation at the of https://www.swarmion.dev/docs/code-structure/services and https://www.swarmion.dev/docs/code-structure/libraries?

**Comment by guillaumeduboc** [comment]
> Can you:
> 
> * add documentation on the `--directory` option for both service and library generators?

I will add this when the `--directory` feature is complete

**Review by fargito — APPROVED** [review]
Thanks a lot! @allcontributors please add @guillaumeduboc for docs

**Pull request merged**